static/panes/device-view.ts

```diff
@@ -180,7 +180,7 @@ export class DeviceAsm extends MonacoPane<monaco.editor.IStandaloneCodeEditor, D
         const selectize = this.selectize;
 
         for (const key in selectize.options) {
-            if (deviceNames.includes(selectize.options[key].name)) {
+            if (!deviceNames.includes(selectize.options[key].name)) {
```

Comment:
whoah that's a fine bug to have fixed. Files under "how did this ever work" ?

Reply:
Not sure. My guess was the compilers that return devices always returned the same device/list of devices, so the select options never needed to be cleared.